creator-node/prod.env

```diff
@@ -1,48 +1,52 @@
 # DO NOT MODIFY these values, use `audius-cli get-config creator-node` for accurate values
 
+# Required node-specific overrides
+creatorNodeEndpoint=
+delegateOwnerWallet=
+delegatePrivateKey=
+spOwnerWallet=
+
+# Infra
 dbUrl=postgres://postgres:postgres@db:5432/audius_creator_node
 dbConnectionPoolMax=500
-
 redisHost=cache
 redisPort=6379
+enableRsyslog=false
 
+# Node.js
+timeout=3600000 # socket inactivity timeout
+headersTimeout=60000
+keepAliveTimeout=5000
+setTimeout=3600000
+
+# Chain
 dataNetworkId=99
 dataProviderUrl=https://poa-gateway.audius.co
 dataRegistryAddress=0xC611C82150b56E6e4Ec5973AcAbA8835Dd0d75A2
 entityManagerAddress=0x2F99338637F027CFB7494E46B49987457beCC6E3
 entityManagerReplicaSetEnabled=false
-disableSnapback=true
-enableRsyslog=false
 ethNetworkId=1
 ethOwnerWallet=0xe886a1858d2d368ef8f02c65bdd470396a1ab188
 ethProviderUrl=https://mainnet.infura.io/v3/a3ed533ddfca4c76ab4df7556e2745e1
 ethRegistryAddress=0xd976d3b4f4e22a238c1A736b6612D22f17b6f64C
 ethTokenAddress=0x18aAA7115705e8be94bfFEBDE57Af9BFc265B998
-headersTimeout=60000
+
+# Application
 identityService=https://identityservice.audius.co
-keepAliveTimeout=5000
+pinAddCIDs=QmXSa6NxfA3e2hCpd8P5gNnZJs9PjZ3yGDHiRHR5B7Rq52,QmNU5Sv8se1aqfUFYFxupX1wdZfGTGnJp6vpoMbLNceGVi
 logLevel=info
+otelTracingEnabled=true
+otelCollectorUrl=https://opentelemetry-collector.audius.co/v1/traces
 maxAudioFileSizeBytes=1000000000
-pinAddCIDs=QmXSa6NxfA3e2hCpd8P5gNnZJs9PjZ3yGDHiRHR5B7Rq52,QmNU5Sv8se1aqfUFYFxupX1wdZfGTGnJp6vpoMbLNceGVi
-setTimeout=3600000
+enforceWriteQuorum=false
+
+# State machine
 stateMonitoringQueueRateLimitInterval=60000
 stateMonitoringQueueRateLimitJobsPerInterval=1
-timeout=3600000
-enforceWriteQuorum=false
-otelTracingEnabled=true
-otelCollectorUrl=https://opentelemetry-collector.audius.co/v1/traces
 maxRecurringRequestSyncJobConcurrency=16
 maxManualRequestSyncJobConcurrency=30
-snapbackHighestReconfigMode=PRIMARY_AND_OR_SECONDARIES
-snapbackUsersPerJob=4000
-contentCacheLayerEnabled=true
-reconfigModePrimaryOnly=true
+# Replica set update
 maxUpdateReplicaSetJobConcurrency=0
-backgroundDiskCleanupDeleteEnabled=true
```

Comment:
this is set to false in creator node config.js

Reply:
ah i think i flipped this to true in PR but for now just added back here good callout